packages/ui/package.json

```diff
@@ -5,5 +5,15 @@
   "license": "MIT",
   "devDependencies": {
     "config": "workspace:*"
+  },
+  "dependencies": {
+    "@babel/core": "^7.17.7",
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.5.1"
```

Comment:
Why aren't these `peerDependencies` as well?


----

For context, say you are using this dependency and some other library that also relies on MUI, e.g. formik-mui. This means if both libraries include MUI, we'll now have 2 MUI libraries bundled together.